### PR TITLE
Engage43 - Move Parameters

### PR DESCRIPTION
**Pull request by jeisenman23**

### CODE REVIEW

**Comment by jgu2** [comment]
Two points here
1. Resolve the conflicts
2. Update pull request title, PR naming pattern `engageAA-BB` where AA is the number of Jira task, BB is a clean PR title. Refer other PRs as example.

File: calliope_app/api/migrations/0066_run_parameter_tab.py
Comment (jmorrisnrel) [inline calliope_app/api/migrations/0066_run_parameter_tab.py:1]: There are two 066 migrations, one for this PR and one for the ratelimit changes. The 3 migration files for this PR will need to be redone to start with 067

File: calliope_app/api/fixtures/admin_run_parameter.json [hunk 1]
```diff
@@ -409,8 +431,9 @@
       "user_visibility": "True",
       "can_evolve": "False",
       "name": "solver",
-      "default_value": "appsi_highs",
-      "choices": "[\"glpk\",\"appsi_highs\",\"amplxpress\",\"xpress_direct\"]"
+      "default_value": "amplxpress",
```
[the remark this answers is not in the document]
Comment (jeisenman23) [reply calliope_app/api/fixtures/admin_run_parameter.json]: Fixed

**Comment by jeisenman23** [comment]
> Two points here @jeisenman23
> 
> 1. Resolve the conflicts
> 2. Update pull request title, PR naming pattern `engageAA-BB` where AA is the number of Jira task, BB is a clean PR title. Refer other PRs as example.

Fixed

**Pull request merged**